.gitignore

```diff
@@ -1,2 +1,8 @@
 node_modules/
 
+# IDE - VSCode
+.vscode/*
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+.history/*
```

Comment:
What has a .history?

Reply:
I think this was generated when I ran something using Angular CLI

(Related: https://stackoverflow.com/questions/63988454/is-it-safe-to-remove-the-history-folder-when-committing-to-git)